nodejs/week1/src/backend/routes/meals.js

```diff
@@ -0,0 +1,33 @@
+const fs = require('fs');
+const filepath = require('path').resolve(__dirname, '..');
+
+
+function getReview(id) {
+    let revData = fs.readFileSync(`${filepath}/data/reviews.json`);
+    let reviews = JSON.parse(revData);
+    const reviewArray = [];
+
+    reviews.forEach(element => {
+        if(element.mealId == id) {
+            reviewArray.push(element);
+        }
+    });
+    return reviewArray;
+}
+
+function mealsRouter() {
+    let mealData = fs.readFileSync(`${filepath}/data/meals.json`);
+    let meals = JSON.parse(mealData);
+    
+    meals.forEach(element => {
+        element["reviews"] = getReview(element.id);

```

Comment:
It's cool to use [] when accessing json keys but you don't really need to do that unless the key name is stored in a var or you're accessing weird key names like numbers, strings with dashes (key-name) and such

```suggestion
        element.reviews = getReview(element.id);

```